example/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright YEAR The Kubernetes Authors.
+Copyright 2017 The Kubernetes Authors.
```

Comment:
it's an example boilerplate, plz exclude this from the pull. thx

Reply:
Done